Tasks/HelmDeployV0/Tests/L0.ts

```diff
@@ -40,6 +41,7 @@ describe("HelmDeployV0 Suite", function () {
         process.env[shared.TestEnvVars.command] = shared.Commands.install;
         process.env[shared.TestEnvVars.chartType] = shared.ChartTypes.Name;
         process.env[shared.TestEnvVars.chartName] = shared.testChartName;
+        process.env[shared.TestEnvVars.version] = shared.testChartVersion;
```

Comment:
You're setting this version for every test then?

Reply:
No. In the `beforeEach` block, the verison input gets deleted.

Reply:
Yes yes. This just means that you're setting a default chartVersion to all test cases right?
Perhaps add a test without sending the chartVersion

Reply:
Since the chartVersion gets deleted before each tests, the subsequent tests are running without a chartVersion. I'm not sure why is a new test required.